gallery/ActivityLogsInsights/Azure Monitor.json

```diff
@@ -7,7 +7,7 @@
 			"name": "Activity Logs Insights",
 			"templates": [
 				{
-					"id": "Workbooks/ActivityLogsInsights/AtScale",
+					"id": "Workbooks/Activity Log/AtScale/",
```

Comment:
Can you also use the existing gallery file as well? activitylog-insights

Reply:
done. I think:)